### DISCUSSION TITLE
Problem disabling rules

### DISCUSSION THREAD

**Discussion by bit1290** [Security-Onion-Solutions/securityonion#6990] [Unsupported Versions]
Hello,

I have da following configuration on /opt/so/saltstack/local/pillar/minions:

[ids_standalone.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7941439/ids_standalone.txt)

As you can see I have disabled the following rules: - 're:ET POLICY', however the do not appear as commented in all.rules:

cat /opt/so/rules/nids/all.rules | grep -v "#" | grep "ET POLICY"
alert tcp $HOME_NET any -> $EXTERNAL_NET 1433 (msg:"ET POLICY Outbound MSSQL Connection to Standard port (1433)"; flow:to_server,established; content:"|12 01 00|"; depth:3; content:"|00 00 00 00 00 00 15 00 06 01 00 1b 00 01 02 00 1c 00|"; distance:1; within:18; content:"|03 00|"; distance:1; within:2; content:"|00 04 ff 08 00 01 55 00 00 00|"; distance:1; within:10; flowbits:set,ET.MSSQL; classtype:bad-unknown; sid:2013410; rev:4; metadata:created_at 2011_08_16, updated_at 2011_08_16;)
alert tcp $HOME_NET any -> $EXTERNAL_NET !1433 (msg:"ET POLICY Outbound MSSQL Connection to Non-Standard Port - Likely Malware"; flow:to_server,established; content:"|12 01 00|"; depth:3; content:"|00 00 00 00 00 00 15 00 06 01 00 1b 00 01 02 00 1c 00|"; distance:1; within:18; content:"|03 00|"; distance:1; within:2; content:"|00 04 ff 08 00 01 55 00 00 00|"; distance:1; within:10; flowbits:set,ET.MSSQL; classtype:bad-unknown; sid:2013409; rev:3; metadata:created_at 2011_08_16, updated_at 2011_08_16;)

After I added this regex I executed the following commands successfully:

sudo salt $SENSORNAME_$ROLE state.apply idstools
sudo so-rule-update

But the rules still dont appear as commented.

Is there something I'm missing?

THANKS :)

**Comment by dougburks** [comment]
The two rules you're showing include flowbits, so you might want to take a look at https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=flowbits#flowbits to see if that is the issue.
